<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spotweb Filter Creator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="spotwebfc.qrc">
    <normaloff>:/images/filter.png</normaloff>:/images/filter.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QPushButton" name="openButton">
     <property name="toolTip">
      <string>Open an existing Spotweb Filter file</string>
     </property>
     <property name="text">
      <string>&amp;Open...</string>
     </property>
     <property name="icon">
      <iconset resource="spotwebfc.qrc">
       <normaloff>:/images/fileload.png</normaloff>:/images/fileload.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>Save the Spotweb filters</string>
     </property>
     <property name="text">
      <string>&amp;Save</string>
     </property>
     <property name="icon">
      <iconset resource="spotwebfc.qrc">
       <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="saveAsButton">
     <property name="toolTip">
      <string>Save the Spotweb filters to another file</string>
     </property>
     <property name="text">
      <string>&amp;Save As...</string>
     </property>
     <property name="icon">
      <iconset resource="spotwebfc.qrc">
       <normaloff>:/images/filesaveas.png</normaloff>:/images/filesaveas.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="fileLineEdit">
     <property name="toolTip">
      <string>Current Spotweb Filter file</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="yourFiltersLabel">
         <property name="text">
          <string>Your Filters</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTreeWidget" name="yourFiltersTreeWidget">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;List of your filters&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;- Add a new filter by clicking the (+) button&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;- Remove a filter by selecting a filter and clicking the (-) button&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="addToolButton">
         <property name="toolTip">
          <string>Add a new filter</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="spotwebfc.qrc">
           <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="removeToolButton">
         <property name="toolTip">
          <string>Remove the selected filter</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="spotwebfc.qrc">
           <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="availableFiltersLabel">
         <property name="text">
          <string>Available Filters</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTreeWidget" name="availableFiltersTreeWidget">
         <property name="toolTip">
          <string>Available filters to choose from in Spotweb</string>
         </property>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="clearAllToolButton">
         <property name="toolTip">
          <string>Uncheck all filters (Clear)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="spotwebfc.qrc">
           <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="spotwebfc.qrc"/>
 </resources>
 <connections/>
</ui>
